radixdlt-java/src/test/java/com/radixdlt/client/util/HashTest.java

```diff
@@ -0,0 +1,26 @@
+package com.radixdlt.client.util;
+
+import com.radixdlt.client.core.util.Hash;
+import okio.ByteString;
+import org.junit.Test;
+
+import java.io.UnsupportedEncodingException;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class HashTest {
+
+    @Test
+    public void testDoubleSha256Hash() {
+        byte[] data = new byte[0];
+        try {
+            data = "Hello Radix".getBytes("UTF-8");
```

Comment:
You could use `StandardCharsets.UTF_8` instead of the string literal to avoid the try-catch block. Because readability.

Furthermore, if you prefer to keep the string literal, I'd much rather let the test case fail, (basically adding a general `throws Exception` clause to the method signature and remove the try + catch blocks).

Reply:
Actually I tried that :D but I failed to import the package, must have been a type, got it working now :) 

Using the StandardCharset results in the `getBytes` not throwing, which is great.